app/build.gradle

```
@@ -72,6 +72,9 @@ android {
             returnDefaultValues = true
             includeAndroidResources = true
         }
+        unitTests.all {
+            systemProperty 'robolectric.logging.enabled', 'true'
+        }        
```

```suggestion
            systemProperty 'robolectric.logging', 'stdout'
        }        
```

Note to reviewer: The PR as stands enables Robolectric debug logging. To get the logs from general Log.* calls you need this line as well.